source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/locale-input/-utils/helpers.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+
+export const hasSearchTerm = (searchTerm: string) => {
+  let lowerSearch = (searchTerm || '').trim().toLocaleLowerCase();
```

Comment:
`searchTerm` is defined as a string but treated as if it is null or undefined here.

Reply:
yeah,
I've changed this to `searchTerm?: string` in the upcoming commit.